tests/test_geodist.py

```
@@ -969,3 +996,184 @@ def test_geodesic_knn_metric():
+
+
+class TestResolveEllipsoid:
+    """Tests for _resolve_ellipsoid helper."""
+
+    def test_wgs84_default(self):
+        a, f = _resolve_ellipsoid("WGS-84")
+        assert a == pytest.approx(6378137.0)
+        assert f == pytest.approx(1.0 / 298.257223563)
+
+    def test_none_returns_wgs84(self):
+        a, f = _resolve_ellipsoid(None)
```

Ruff (0.15.2)

[warning] Unpacked variable `f` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)